train/tr11-176B-ml/smaller_models/tr11e-350M-ml.slurm

```diff
@@ -104,7 +104,7 @@ GPT_ARGS=" \
     --seq-length $SEQ_LEN \
     --max-position-embeddings $SEQ_LEN \
     --micro-batch-size $MICRO_BATCH_SIZE \
-    --rampup-batch-size 192 16 9_765_625 \
+    --rampup-batch-size 192 32 9_765_625 \
```

Comment:
I'm just taking the opportunity to raise a change here that doesn't seem to me to be related to change `@gpu` to `@v100` (especially if it's not a desirable change)

Reply:
ah it's a desirable change, the 16-increment breaks a multiple-of-32 assumption and it should indeed be 32. I should have merged this earlier though. I'm a bit lazy to make it a separate PR but if someone thinks it's important I'll separate it

Reply:
thanks for spotting it!

Reply:
Thanks for the explanation :smile: 
Fine with me to keep it here!